content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -66,98 +67,84 @@ pod/nginx-701339712-e0qfq   1/1           Running       0          35s
 
 ## Test the service by accessing it from another pod
 
-You should be able to access the new `nginx` service from other pods. To test, access the service from another pod in the default namespace. Make sure you haven't enabled isolation on the namespace.
+You should be able to access the new `nginx` Service from other Pods. To test, access the Service from another Pod in the `default` namespace. Make sure you haven't enabled isolation on the namespace.
```

Comment:
Line 68: Capitalize Pod, 
Test the service by accessing it from another Pod

Line 70 - 71, clean up: To access the `nginx` Service from another Pod in the `default` namespace,
start a busybox container:

Reply:
I capitalized Service on line 70-71; however, I am not completely convinced in these examples it is necessary.

Reply:
I agree.